SimpleMvvmToolkit.Samples.Messaging.Core/SimpleMvvmToolkit.Samples.Messaging.Core/ViewModels/CustomerListViewModel.cs

```diff
@@ -0,0 +1,117 @@
+﻿using System.Collections.ObjectModel;
```

Comment:
Update to add DI code as in [SimpleMvvmToolkit.Samples.WPF.Core](https://github.com/SimpleMvvm/SimpleMvvmToolkit.Samples.WPF.Core).